google/cloud/forseti/notifier/notifiers/email_violations.py

```diff
@@ -161,16 +170,30 @@ def _send(self, **kwargs):
         content = notification_map['content']
         attachment = notification_map['attachment']
 
-        try:
-            self.mail_util.send(
-                email_sender=self.notification_config['sender'],
-                email_recipient=self.notification_config['recipient'],
-                email_subject=subject,
-                email_content=content,
-                content_type='text/html',
-                attachment=attachment)
-        except util_errors.EmailSendError:
-            LOGGER.warn('Unable to send Violations email')
+        if self.notifier_config.get('email_connector_config'):
+            try:
+                self.connector.send(
+                    email_sender=self.notifier_config
+                    .get('email_connector_config').get('sender'),
+                    email_recipient=self.notifier_config
+                    .get('email_connector_config').get('recipient'),
+                    email_subject=subject,
+                    email_content=content,
+                    content_type='text/html',
+                    attachment=attachment)
+            except util_errors.EmailSendError:
+                LOGGER.warn('Unable to send Violations email')
+        else:
+            try:
```

Comment:
Make comments here on which is for backward compatibility so that they are easy to remove in the future.  Please do this everywhere.

Reply:
Sure